_posts/2017-11-16-tenser's-transformation.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Tenser’s Transformation"
+date:   2017-11-16
+sources: [Xanathar.168]
+tags: [wizard, level6, transmutation]
+---
+
+**6th-level transmutation**
+
+**Casting Time**: 1 action
+
+**Range**: Self
+
+**Components**: V, S, M (a few hairs from a bull)
+
+**Duration**: Concentration, up to 10 minutes
+
+You endow yourself with endurance and martial prowess fueled by magic. Until the spell ends, you can’t cast spells, and you gain the following benefits: 
+
+- You gain 50 temporary hit points. If any of these remain when the spell ends, they are lost. 
+- You have advantage on attack rolls that you make with simple and martial weapons. 
+- When you hit a target with a weapon attack, that tar- ; i get takes an extra 2d 12 force damage. 
```

Comment:
Now this is an interesting typo. I don't know how it happened, but it should say "that target takes"

Reply:
Also, `2d12` does not have a space in it.

Reply:
copy and paste gives some very strange errors sometimes.